src/utils/fs.rs

```diff
@@ -72,6 +72,7 @@ impl TempFile {
             .read(true)
             .write(true)
             .create(true)
+            .truncate(false)
```

Comment:
This line [keeps the old behavior](https://rust-lang.github.io/rust-clippy/master/index.html#/suspicious_open_options) and satisfies Clippy. I would have thought that `truncate(true)` would make more sense, but it causes our tests to fail.

Reply:
We will investigate further in #1989